tshirts2022.py

```diff
@@ -0,0 +1,29 @@
+import json
+import re
+from urllib import request
+
+URL = "https://www.pycon.jp/support/bootcamp.html"
+with request.urlopen(URL) as f:
+    html = f.read().decode("utf-8")
+pattern = re.compile(r"pyconjp.*/event/(\d+)/(?!.*中止)")
+ids = ",".join(m[1] for m in pattern.finditer(html))
+
+connpass_api = "https://connpass.com/api/v1/event/"
+url = f"{connpass_api}?event_id={ids}&count=100"
+with request.urlopen(url) as f:
+    data = json.loads(f.read())
```

Comment:
`json.load()` で良いのでは？

```suggestion
with request.urlopen(url) as f:
    data = json.load(f)
```

Reply:
たしかに

Reply:
1行にまとめた
```
data = json.load(request.urlopen(url))
```